test/library/metrics/query.go

```diff
@@ -38,26 +37,15 @@ func NewPrometheusClient(ctx context.Context, kclient kubernetes.Interface, rc r
 		return nil, fmt.Errorf("failed to get thanos-querier route: %w", err)
 	}
 	host := route.Status.Ingress[0].Host
-
-	var bearerToken string
-	secrets, err := kclient.CoreV1().Secrets("openshift-monitoring").List(ctx, metav1.ListOptions{})
+	expirationSeconds := int64(24 * time.Hour / time.Second)
+	req, err := kclient.CoreV1().ServiceAccounts("openshift-monitoring").CreateToken(ctx, "prometheus-k8s",
```

Comment:
Is the constant `prometheusServiceAccountName` unused now?

Reply:
Deleted.